src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -81,12 +81,14 @@ internal async Task EnsureInitializedAsync(CancellationToken cancellationToken)
                     if (p.IsSearchable != tempStatus.IsSearchable ||
                         p.IsSupported != tempStatus.IsSupported ||
                         p.IsPartiallySupported != tempStatus.IsPartiallySupported ||
-                        p.SortStatus != result.SortStatus)
+                        p.SortStatus != result.SortStatus ||
+                        p.SearchParameterStatus != result.Status)
```

Comment:
Any way to add a test? https://github.com/microsoft/fhir-server/blob/323b9e8b7d40ad9fbc14f0fb0a54c5657d306416/src/Microsoft.Health.Fhir.Core.UnitTests/Features/Search/Registry/SearchParameterStatusManagerTests.cs

Reply:
Updated a unit test to cover this case.